media/js/firefox/welcome-page2.js

```diff
@@ -13,23 +13,29 @@ if (typeof window.Mozilla === 'undefined') {
     var PocketButton = {};
 
     PocketButton.init = function(){
+        var buttons;
 
-        var pocketButton = document.getElementById('pocket-cta');
+        // Collect all monitor buttons
+        buttons = document.getElementsByClassName('js-pocket-button');
```

Comment:
Not a blocker for this PR and I'm not saying you need to refactor it here - but this seems almost identical to the monitor button logic. We should probably be making this more DRY. Let's save that for a future issue though?

Reply:
+100 agree. I was thinking that perhaps we can refactor the monitor JS to be usable by all fxa-product ctas?
We had originally separated it out because we were unsure if the pocket button should be made into it's own macro.

Reply:
Filed issue for this: #7985